tripal/src/Plugin/TripalIdSpace/TripalDefaultIdSpace.php

```diff
@@ -0,0 +1,671 @@
+<?php
+
+namespace Drupal\tripal\Plugin\TripalIdSpace;
+
+use Drupal\tripal\TripalVocabTerms\TripalIdSpaceBase;
+use Drupal\tripal\TripalVocabTerms\TripalTerm;
+
+/**
+ * Default Implementation of TripalIdSpaceBase
+ *
+ *  @TripalIdSpace(
+ *    id = "tripal_default_id_space",
+ *    label = @Translation("Deafult Tripal IdSpace"),
+ *  )
+ */
+class TripalDefaultIdSpace extends TripalIdSpaceBase {
+  /**
+   * A simple boolean to prevent Chado queries if the ID space isn't valid.
+   *
+   * @var bool
+   */
+  protected $is_valid = FALSE;
+
+  /**
+   * {@inheritdoc}
+   */
+  public function __construct(array $configuration, $plugin_id, $plugin_definition) {
+    parent::__construct($configuration, $plugin_id, $plugin_definition);
+
+    // Instantiate the TripalLogger
+    $this->messageLogger = \Drupal::service('tripal.logger');
+
+  }
+
+  /**
+   * {@inheritDoc}
+   */
+  public function getParent($child) {
+  }
```

Comment:
We should let someone know that this isn't supported in case they do try to use this implementation.

```suggestion
    $this->messageLogger->warning('TripalDefaultIdSpace: Relationships between terms are currently not supported by this implementation.');
    return NULL;
  }
```

Reply:
I saw below that the getChildren is implemented... why isn't this one? it looks like it would be relatively easy to grab the parents since they are stored directly.

Reply:
This is not implemented in the ChadoVocabulary plugin either.    We need to implement it in both.  For now can we just add an issue to remind us to do both?